src/data/clients/requirementsClient.ts

```diff
@@ -7,8 +7,14 @@ export interface CreateRequirementInput {
   name: string
   incomeProfile: any // Replace 'any' with the actual type for incomeProfile
```

Comment:
Maybe this type will come from enum?

Reply:
i was thinking to do change all the types once we are done with the mortgage section, when you'll work on the business logic i can work on writing correct types on every file

Reply:
Okay no problem!